ts/receiver/queuedJob.ts

```diff
@@ -470,7 +464,7 @@ async function handleRegularMessage(
     );
   }
 
-  if (source !== ourNumber) {
+  if (source !== ourNumber && primarySource) {
     message.set({ source: primarySource });
```

Comment:
Actually we were already passing `primarySource.key` one level above. However, I changed the method signature to use `PubKey` instead of any `any` and updated the caller.